test/MetrcSalesTest.js

```diff
@@ -0,0 +1,176 @@
+
+
```

Comment:
this file is essentially really good. Composition used where possible, makes it very readable.

The only feedback I could have, is that tests for negative paths, like error handling, validation, seem to be missing.